news/2023/2023-07-28-flag-change-updates.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Country Flag Change Updates"
```

Comment:
i think this title sounds a bit weird... how about "Enabling Country Flag Changes"?